dojo/unittests/dojo_test_case.py

```diff
@@ -441,6 +441,49 @@ def import_scan_with_params(self, filename, scan_type='ZAP Scan', engagement=1,
 
         return self.import_scan(payload, expected_http_status_code)
 
+    def import_checkmarx_scan_with_params(self, filename, scan_type='Checkmarx Scan', engagement=1, minimum_severity='Low', active=True, verified=True,
```

Comment:
sorry for barking in on a draft PR, but I think you could just use the existing import_scan_with_params and provide the scan_type parameter?

Reply:
you're right :)